utils/_context/_scenarios.py

```diff
@@ -1716,7 +1716,11 @@ def all_endtoend_scenarios(test_object):
 
     appsec_auto_events_extended = EndToEndScenario(
         "APPSEC_AUTO_EVENTS_EXTENDED",
```

Comment:
Will it make more sense to create a new scenario for the `anonymization` mode  (e.g. `APPSEC_AUTO_EVENTS_ANONYMIZATION`?

Mixing `extended` and `anonymization` makes no sense as the latter will enforce safe mode.

Reply:
This prevents from adding a new scenario (They cost a lots of time) and allows to check that during deprecation phase, this the latter that is used in case both are set.
So I think this is a win win, except for the name of the scenario, that can be changed later.